fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tag/v1/CommonItemTags.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.v1;
+
+import net.minecraft.item.Item;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.common.TagRegistration;
+
+/**
+ * See {@link net.minecraft.tag.ItemTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public class CommonItemTags {
```

Comment:
What about nuggets, blocks, raw ores and raw ore blocks?

See also https://github.com/AztechMC/Modern-Industrialization/blob/15e36c35c54ec8ddd066c8533a24f8c1d12b93d2/src/main/java/aztech/modern_industrialization/materials/MIMaterials.java#L469.

Reply:
Added